server/plugin/api.go

```diff
@@ -359,9 +375,85 @@ func (p *Plugin) handleSubscriptionNotifications(w http.ResponseWriter, r *http.
 		return
 	}
 
-	attachment := &model.SlackAttachment{
-		Text: body.DetailedMessage.Markdown,
+	var attachment *model.SlackAttachment
+	switch body.EventType {
+	case constants.WorkItemCreated, constants.WorkItemUpdated, constants.WorkItemDeleted, constants.WorkItemCommented:
+		attachment = &model.SlackAttachment{
+			Text: body.DetailedMessage.Markdown,
+		}
+	case constants.PullRequestCreated, constants.PullRequestUpdated, constants.PullRequestMerged:
+		reviewers := p.getReviewersListString(body.Resource.Reviewers)
+
+		attachment = &model.SlackAttachment{
+			Pretext: body.Message.Markdown,
+			Title:   fmt.Sprintf("%d: %s", body.Resource.PullRequestID, body.Resource.Title),
+			Fields: []*model.SlackAttachmentField{
+				{
+					Title: "Target Branch",
+					Value: strings.Split(body.Resource.TargetRefName, "/")[2],
```

Comment:
i think here we should check whether this array is actually of length 2 or not because it can panic here